### ISSUE TITLE
clipboard-copy field argument not working (or unclear)

### ISSUE THREAD

**Issue by theOvercloud** [Keeper-Security/Commander#1328]
I want to use the clipboard-copy command to get a filed value from the vault. I am using the following command:

`keeper clipboard-copy --field host --output stdout "<path to entry>"`

If i use the command without the `field` argument, i get the password of the entry in stdout as expected. However, as soon as I add the filed argument, the output is empty. I tried a few fields, but non worked. I used the names i get from the `get --format json` output. Am I doing something wrong?

Thanks in advance!

**Comment by sk-keeper** [comment]
You are correct. The `clipboard-copy` command does not work with compound fields.
The `host` contains two values: hostname and port.

This issue will be fixed in the next release.
Thank you for letting us know about this issue.

**Comment by theOvercloud** [comment]
Thanks for the quick reply!

Just to clarify, the --field flag should be able to get any field from an entry, correct? In my case, I would like to get a `keypair` field.

**Comment by sk-keeper** [comment]
Commander v16.11.20 has been published
It supports compound fields, the fields that contain multiple properties
`host` field is defined as `{'hostName': '', 'port': ''}`
```
My Vault> rti -lf keyPair
Field Type ID    Type        Value Type    Value Format
---------------  ----------  ------------  -----------------------------------
keyPair          privateKey  object        {'publicKey': '', 'privateKey': ''}
```
`clipboard-copy --field=host` returns a string representation of the field
for `host` it is `hostName:poort`
for `keyPair` it is private key `privateKey` only

To access to the specific property of the compound field add property name after a colon
```
clipboard-copy --field=keyPair:pr
```
the property name does not have to be a full property name.


`keyPair` needs to be case sensitive

**Comment by theOvercloud** [comment]
Can confirm works as described. Thanks!
